src/components/biggive-campaign-card-filter-grid/biggive-campaign-card-filter-grid.tsx

```diff
@@ -68,22 +68,22 @@ export class BiggiveCampaignCardFilterGrid {
   /**
    * JSON array of category key/values, or takes a stringified equiavalent (for Storybook)
    */
-  @Prop() categoryOptions: string | Record<string, string>;
+  @Prop() categoryOptions: string | Record<string, string> | string[];
 
   /**
    * JSON array of beneficiary key/values, or takes a stringified equiavalent (for Storybook)
    */
-  @Prop() beneficiaryOptions: string | Record<string, string>;
+  @Prop() beneficiaryOptions: string | Record<string, string> | string[];
```

Comment:
donate is already passing in 'string[]', at e.g. https://github.com/thebiggive/donate-frontend/blob/4f1c1b691dbf509c3f24799273c2fa5f08b283e0/src/app/explore/explore.component.html#L15 . Sadly we don't have type checking at that point so it didn't match the declaration here.